### PR TITLE
[8.0][ADD] New module account_locked_line_search

### PR DESCRIPTION
**Pull request by kenvac**
## Description
OCA community has a module which locks account move. There is no way to search account move line of lock move. 
This module fills the gap and adds a related field on account.move.line object. This enables a user to search account.move.line of locked moves

## Version
The proposed PR is only tested on V8.0

### CODE REVIEW

File: account_renumber/test/create_moves.py [hunk 1]
```diff
@@ -94,6 +94,7 @@ def create_lots_of_account_moves(dbname, user, passwd, howmany):
 # ------------------------------------------------------------------------
 # ------------------------------------------------------------------------
 
+
```
Comment (sbidoul) [inline account_renumber/test/create_moves.py]: this change should not behere

**Review by sbidoul — COMMENTED** [review]
Thanks for this contribution. I'm not entirely comfortable with the module name. If the goal is searching, perhaps the module should be best named `account_locked_line_search`?

File: account_renumber/test/create_moves.py (hunk 1 above)
Comment (kenvac) [reply account_renumber/test/create_moves.py]: If I remove this change it is breaking travis build violating pep E305 convention.

**Comment by kenvac** [comment]
Thanks for reviewing my work @sbidoul. I've made all changes recommended by you in above comments.

**Review by sbidoul — APPROVED** [review]
Thanks!

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**